src/operator/every.ts

```diff
@@ -18,22 +16,21 @@ export function every<T>(predicate: (value: T, index: number, source: Observable
                          thisArg?: any): Observable<boolean> {
   const source = this;
   if (source._isScalar) {
-    const result: boolean = tryCatch(predicate).call(thisArg || this, source.value, 0, source);
-    if (result === errorObject) {
-      return new ErrorObservable(errorObject.e, source.scheduler);
-    } else {
+    try {
```

Comment:
You should isolate the try/catch to it's own function. Think of it this way: All code that is in the function that contains the try/catch is unoptimized... so you want most of the code to live where try/catch does not.